opm/autodiff/MultisegmentWells.hpp

```diff
@@ -237,6 +237,10 @@ namespace Opm {
 
             const Vector& wellPerfEfficiencyFactors() const;
 
+            // just return the passed well state
+            template<class WellState>
+            const WellState& wellState(const WellState& well_state) const { return well_state; }
```

Comment:
I do not understand the use of this function. Could you explain?

Reply:
this is currently only needed for restart. Primarily because we did not want to open that can of worms in the same PR.

Reply:
Ok, but what is the pattern here? I see that it's probably related to the small change in the output code, but I don't understand the need for this?

Reply:
This is the same trick that is used by the output code. Instead of changing the interface of the solver and timestepping classes we just add a trivial method for the legacy code.